foundation/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "langium-website-foundation",
```

Comment:
Foundation sounds like an organization, core might be better here.

Reply:
Changed to `core` in next commit ⬇️